rclcpp/src/rclcpp/executors/executor_notify_waitable.cpp

```diff
@@ -156,9 +160,13 @@ ExecutorNotifyWaitable::add_guard_condition(rclcpp::GuardCondition::WeakPtr weak
   auto guard_condition = weak_guard_condition.lock();
   if (guard_condition && notify_guard_conditions_.count(weak_guard_condition) == 0) {
     notify_guard_conditions_.insert(weak_guard_condition);
-    if (on_ready_callback_) {
-      guard_condition->set_on_trigger_callback(on_ready_callback_);
+    guard_condition->set_on_trigger_callback([this](size_t cnt){
```

Comment:
nitpick: use full words

```suggestion
    guard_condition->set_on_trigger_callback([this](size_t count){
```